packages/front-end/components/MetricAnalysis/MetricAnalysis.tsx

```diff
@@ -132,11 +134,17 @@ function getLookbackSelected(lookbackDays: number): string {
     : `custom`;
 }
 
-function getDesiredSettings(
-  id: string,
-  values: MetricAnalysisFormFields,
-  endOfToday: Date
-): CreateMetricAnalysisProps {
+export function getMetricAnalysisProps({
```

Comment:
Exporting multiple functions from a component file causes HMR (hot module reloading) to stop working on dev.  You might have noticed it causing the entire page to refresh on every change.  Not a big deal, but better to move this to a separate util/services file instead and import it both places it is used.

Reply:
Thanks, moved it to a util file in the same metric analysis folder.